student.rb

```diff
@@ -0,0 +1,11 @@
+# student with there classroom and skills
+class Student < Person
+  def initialize(class_room)
+    super()
+    @class_room = class_room
+  end
```

Comment:
- [ ] The other parameters for creating a person needs to be provided when creating a student. Kindly add them.

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```